app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,64 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun UserScoreProgressBar (
+    score: Float,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = Modifier
+            .size(50.dp)
+            .padding(2.dp)
+    ) {
+        Canvas(
+            modifier = Modifier.size(50.dp)
+        ) {
+            drawArc(
+                Color(red=212, green =255,blue=212),
```

Comment:
define color in theme/Color.kt and then call the defined color here. 
Also it is a convention to define a color with hex value (you can see how others are defined in Color.kt file). You don't have to change it her but moving forward should try to stick to that convention.